blender_imgui.py

```diff
@@ -18,6 +18,8 @@
 #
 # ##### END GPL LICENSE BLOCK #####
 
+blender_imgui_global_framerate_cap = 10
+"I use this to limit the amount of frames that blender_imgui can emit, it helps to offload stress of the main thread in some situations, like when running modals."
```

Comment:
Could you turn this into an actual comment prior to the definition of the global variable? (unless this is idiomatic but idk)

Reply:
Hi, typing out string literals like this under a symbol/object sets the __doc__ "doc string" child member in Python. So its kinda like Doxygen standards but for python

If you prefer regular comments though feel free to change it.

https://pandas.pydata.org/docs/development/contributing_docstring.html

It never gets recognized as "Docstrings" if its just a regular comment.
![image](https://user-images.githubusercontent.com/20190653/161732393-9cfe3cba-831a-4826-b361-0f4cd281e030.png)
![image](https://user-images.githubusercontent.com/20190653/161732879-16e05420-1370-41ad-82c6-3a273be3402c.png)

Reply:
Hi, this is sets the __doc__ member variable of the object, its a "doc string". So its sort of like doxygen comments standards in  the C++ space but but for Python.

https://pandas.pydata.org/docs/development/contributing_docstring.html

I use it just because I like how it then shows the comments as a popup when hovering the variable, but you can change it into a # comment if you like

The downside of # comments is that the __doc__ member var wont get set & things like hover over popup comments wont show.
![image](https://user-images.githubusercontent.com/20190653/161735292-480b6dd5-f7bd-49d6-a670-c6bcf87a3781.png)
![image](https://user-images.githubusercontent.com/20190653/161735308-468f11d2-753e-4b46-b08d-e9ffeedff3c4.png)